app/src/components/atoms/Chips/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import { Pressable, Text } from 'react-native';
+import { Icon } from 'react-native-elements';
+import colors from '../../../../colors';
+
+export const Chips = ({
+    title,
+    icon,
+    hiddenIcon = false,
+    onPress,
+    elevated = false,
+    type,
+    customStyle,
+    disabled = false,
+    selected = false,
+}) => {
+    const [isSelected, setIsSelected] = useState(false);
+    const chipColor =
+        isSelected || selected ? 'bg-secondary-500 border-0' : 'bg-white';
```

Comment:
Será que aqui não faz sentido iniciar o isSelected com o valor do selected? Para gente não ter que fazer essa dupla verificação desses 2 campos.
```suggestion
    const [isSelected, setIsSelected] = useState(selected);
    const chipColor =
        selected ? 'bg-secondary-500 border-0' : 'bg-white';
```

Reply:
feito